### PR TITLE
Make container ID as optional prop

### PR DESCRIPTION
**Pull request by luisguerra**
This PR enables the ability to modify the div to avoid collision with existing ID's or define multiple elements

### CODE REVIEW

**Comment by luisguerra** [comment]
cc @francineemilia @icaldana @AdaltonLeite @AleFossati @lemazzo

**Comment by AleFossati** [comment]
Olá @luisguerra, assim que possível faremos a revisão e teste das alterações.
Obrigado pela contribuição!

File: src/bricks/brand/index.tsx [hunk 1]
```diff
@@ -24,7 +24,12 @@ import { TBrand } from './types';
  *
  * @see {@link https://www.mercadopago.com.ar/developers/en/docs/checkout-bricks/brand-brick/introduction Brand Brick documentation} for more information.
  */
-const Brand = ({ onReady = onReadyDefault, customization, locale }: TBrand) => {
+const Brand = ({
+  onReady = onReadyDefault,
+  customization,
+  locale,
+  divId = 'brandBrick_container',
```
Comment (AdaltonLeite) [inline src/bricks/brand/index.tsx:31]:
```suggestion
  id = 'brandBrick_container',
```

File: src/bricks/brand/index.tsx [hunk 2]
```diff
@@ -50,7 +55,7 @@ const Brand = ({ onReady = onReadyDefault, customization, locale }: TBrand) => {
       window.brandBrickController?.unmount();
     };
   }, [customization, onReady]);
-  return <div id="brandBrick_container"></div>;
+  return <div id={divId}></div>;
```
Comment (AdaltonLeite) [inline src/bricks/brand/index.tsx:58]:
```suggestion
  return <div id={id}></div>;
```

File: src/bricks/brand/brand.test.tsx [hunk 3]
```diff
@@ -26,4 +26,20 @@ describe('Test Brand Brick Component', () => {
...
+  test('should found the id of Brand Brick div if specified', async () => {
+    MercadoPagoInstance.publicKey = 'PUBLIC_KEY';
+    const element = await render(
+      <Brand
+        customization={{
+          text: {
+            valueProp: 'payment_methods_logos',
+          },
+        }}
+        divId="customBrandBrick_container"
```
Comment (AdaltonLeite) [inline src/bricks/brand/brand.test.tsx:38]:
```suggestion
        id="customBrandBrick_container"
```

File: src/bricks/brand/index.tsx [hunk 4]
```diff
@@ -37,7 +42,7 @@ const Brand = ({ onReady = onReadyDefault, customization, locale }: TBrand) => {
         },
       },
       name: 'brand',
-      divId: 'brandBrick_container',
+      divId,
```
Comment (AdaltonLeite) [inline src/bricks/brand/index.tsx:45]:
```suggestion
      divId: id,
```

File: src/bricks/brand/types.ts [hunk 5]
```diff
@@ -18,6 +18,10 @@ export type TBrand = {
    * @see {@link https://www.mercadopago.com/developers/en/docs/checkout-bricks/additional-content/select-language Bricks language customization} documentation.
    */
   locale?: 'es-AR' | 'es-CL' | 'es-CO' | 'es-MX' | 'es-VE' | 'es-UY' | 'es-PE' | 'pt-BR' | 'en-US';
+  /**
+   * Optional. Container ID where the Brick will be rendered. Default: 'walletBrick_container'
+   */
+  divId?: string;
```
Comment (AdaltonLeite) [inline src/bricks/brand/types.ts:24]:
```suggestion
  id?: string;
```

**Comment by luisguerra** [comment]
(Sorry I don't speak Portuguese 😅 , hello From Mexico, so I will write in English)
Thanks for feedback @AdaltonLeite, makes sense. I have made the requested changes

File: src/bricks/cardPayment/type.ts [hunk 6]
```diff
@@ -100,6 +100,10 @@ export type TCardPayment = {
      */
     visual?: object;
   };
+  /**
+   * Optional. Container ID where the Brick will be rendered. Default: 'walletBrick_container'
```
Comment (AleFossati) [inline src/bricks/cardPayment/type.ts:104]:
```suggestion
   * Optional. Container ID where the Brick will be rendered. Default: 'cardPaymentBrick_container'
```

File: src/bricks/payment/type.ts [hunk 7]
```diff
@@ -97,6 +97,10 @@ export type TPaymentType = {
    * @see {@link https://www.mercadopago.com/developers/en/docs/checkout-bricks/additional-content/select-language General Customizations # Select Language} documentation.
    */
   locale?: string;
+  /**
+   * Optional. Container ID where the Brick will be rendered. Default: 'walletBrick_container'
```
Comment (AleFossati) [inline src/bricks/payment/type.ts:101]:
```suggestion
   * Optional. Container ID where the Brick will be rendered. Default: 'paymentBrick_container'
```

File: src/bricks/statusScreen/types.ts [hunk 8]
```diff
@@ -28,6 +28,10 @@ export interface IStatusScreenBrickSettings extends IStatusScreenBrickCallbacks
    * @see {@link https://www.mercadopago.com/developers/en/docs/checkout-bricks/additional-content/select-language General Customization # Select Language} documentation.
    */
   locale?: 'es-AR' | 'es-CL' | 'es-CO' | 'es-MX' | 'es-VE' | 'es-UY' | 'es-PE' | 'pt-BR' | 'en-US';
+  /**
+   * Optional. Container ID where the Brick will be rendered. Default: 'walletBrick_container'
```
Comment (AleFossati) [inline src/bricks/statusScreen/types.ts:32]:
```suggestion
   * Optional. Container ID where the Brick will be rendered. Default: 'statusScreenBrick_container'
```

**Review by AleFossati — CHANGES_REQUESTED** [review]
Hello @luisguerra,

I have tested the changes and it looks great, there's only a few changes needed, it seems that the default values in the comments are all referencing wallet brick.

After this fix, I think that we are ready to merge!

**Comment by icaldana** [comment]
@luisguerra hey, could you please check @AleFossati comments? Thank you a lot!

**Review by icaldana — APPROVED** [review]

**Review by AleFossati — APPROVED** [review]

**Pull request merged**